.github/workflows/translations.yml

```diff
@@ -1,66 +1,87 @@
-name: Update translation source
+name: Update Translation Source
 
 on:
   workflow_dispatch:
   schedule:
-    # runs once per month
+    # runs at the start of each month (UTC)
     - cron: '0 0 1 * *'
+  pull_request:
+    paths:
+      - '.github/workflows/translations.yml'
 
 jobs:
   translations:
     # Do not run the scheduled workflow on forks
     if: github.event_name != 'schedule' || github.repository_owner == 'Cockatrice'
 
+    name: Update translation source
     runs-on: ubuntu-latest
 
     steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
       - name: Install lupdate
         shell: bash
         run: |
           sudo apt-get update
           sudo apt-get install -y --no-install-recommends qttools5-dev-tools
```

Comment:
Does it matter if we use qt5 or 6 here?

Reply:
That's a good question. Probably good to use Qt6 here as well to stay uniform in the repo with the recent changes.

Reply:
I tried to get `lupdate` to work in a quick test - no luck.
So Qt5 is good enough for now I guess